### ISSUE TITLE
Unable to resolve action

### ISSUE THREAD

**Issue by xobotyi** [fastify/github-action-merge-dependabot#17]
## 🐛 Bug Report

Github workflows are unable to resolve v2 with error:
```
Unable to resolve action `fastify/github-action-merge-dependabot@v2`, unable to find version `v2`
```

## To Reproduce

Using config below:
```yaml
dependabot-merge:
  name: "Dependabot automerge"
  needs: test
  runs-on: ubuntu-latest
  if: github.actor == 'dependabot[bot]' && github.event_name == 'pull_request'
  steps:
    - uses: fastify/github-action-merge-dependabot@v2
      with:
        github-token: ${{secrets.GITHUB_TOKEN}}
```

## Expected behavior

Action works

**Comment by simoneb** [comment]
use one of the published versions: v2.0.0

**Comment by xobotyi** [comment]
@simoneb just worked it out =)
So there were issue with docs (now fixed)
